Fw/Buffer/Buffer.cpp

```diff
@@ -35,11 +35,19 @@ Buffer::Buffer(const Buffer& src) : Serializable(),
 {}
 
 Buffer::Buffer(U8* data, U32 size, U32 context) : Serializable(),
+#if  data == nullptr && size == 0
```

Comment:
I don't think you can you `#if` in this way (to access realtime data) and if you can, it would violate the C++ coding practice of keeping compiler directives simple.

I think a cleaner fix would be to always use `m_serialize_repr()` in the initializer list, and then add the following to the constructor code block:

```
{
     if (m_bufferData != NULL) {
        this->m_serialize_repr.setExtBuffer(m_bufferData, m_size);   
    }
}
```

-- or --

```
{
    this->setData(m_bufferData);
}
```

Reply:
Looks fixed.  I am letting this run through CI